cg/cli/backup.py

```diff
@@ -34,6 +40,42 @@ def backup(context: CGConfig):
     pass
 
 
+@backup.command("flow-cells")
+@DRY_RUN
+@click.pass_obj
+def backup_flow_cells(context: CGConfig, dry_run: bool):
+    """Backup flow cells."""
+    pdc_api = PdcAPI(binary_path=context.pdc.binary_path, dry_run=dry_run)
+    status_db: Store = context.status_db
+    for flow_cell in get_flow_cells_from_path(flow_cells_dir=Path(context.flow_cells_dir)):
```

Comment:
I think I'd extract the `get_flow_cells_from_path` call to a variable

Reply:
Fixed